clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/provider/agent/GoogleBackendServiceCachingAgent.groovy

```diff
@@ -67,10 +67,12 @@ class GoogleBackendServiceCachingAgent extends AbstractGoogleCachingAgent {
     if (globalBackendServices) {
       ret.addAll(globalBackendServices.collect { toGoogleBackendService(it, GoogleBackendService.BackendServiceKind.globalBackendService) })
     }
+    def regions = credentials.regions.collect { it.name }
+
     timeExecute(compute.regions().list(project),
                 "compute.regions.list",
                 TAG_SCOPE, SCOPE_GLOBAL
-    ).items.each { Region region ->
+    ).items.findAll { regions.contains(it.getName()) }.each { Region region ->
```

Comment:
+1, seems like we can get rid of the live call as long as we compute the regions we care about up front in the credentials.